src/test/java/xyz/thepathfinder/android/MyClusterListener.java

```diff
@@ -0,0 +1,144 @@
+package xyz.thepathfinder.android;
+
+
+import com.google.gson.JsonObject;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+public class MyClusterListener implements ClusterListener{
```

Comment:
I don't like the fact that I have to override 100 methods to extend ClusterListener